api/services/issue_service.py

```diff
@@ -0,0 +1,15 @@
+from api.app import db
+from api.models.issue import Issue
+
+
+def save_new_issue(title, preview, message, sources):
+    issue = Issue(title=title, preview=preview, message=message, sources=sources)
```

Comment:
I think we can skip setting these args manually because of pythons magical `**kwargs` thing.

I'm pretty sure that were gonna take in new issues as json objects in a post endpoint. So we should be able to just parse that json into a `dict` then pass that dict into `Issue()` and kwargs will automatcally unpack the dict and set the attributes for us.